expression/builtin_vectorized.go

```diff
@@ -14,13 +14,97 @@
 package expression
 
 import (
+	"sync"
+
 	"github.com/pingcap/errors"
+	"github.com/pingcap/parser/mysql"
 	"github.com/pingcap/tidb/sessionctx"
 	"github.com/pingcap/tidb/types"
 	"github.com/pingcap/tidb/types/json"
 	"github.com/pingcap/tidb/util/chunk"
 )
 
+// columnBufferAllocator is used to allocate and release column buffer in vectorized evaluation.
+type columnBufferAllocator interface {
+	// alloc allocates a column buffer with the specific eval type and capacity.
+	// the allocator is not responsible for initializing the column, so please initialize it before using.
+	alloc(evalType types.EvalType, capacity int) (*chunk.Column, error)
```

Comment:
can we rename these two fields to `put` and `get`?

Reply:
These two methods are renamed.